### PR TITLE
[PLAYER-1678] Fixed data selector items ref issue, added unit tests

### PR DESCRIPTION
**Pull request by jpfranco**
This PR is mostly for the bug fix although it already has some of the unit tests. I will create yet another PR with the remaining unit tests.

### CODE REVIEW

**Comment by pilievOoyala** [comment]
LGTM

File: tests/components/dataSelector-test.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  function findItemByLabel(label) {
+    for (var i = 0; i < items.length; i++) {
+      if (items[i].props.ariaLabel === label) {
+        return items[i];
+      }
+    }
+    return null;
+  }
+
+  beforeEach(function() {
```
Comment (aeng7) [inline tests/components/dataSelector-test.js]: Should we reset "component" here as well?

File: tests/views/contentScreen-test.js [hunk 2]
```diff
@@ -34,18 +48,27 @@ describe('ContentScreen', function () {
...
     var contentScreen = ReactDOM.findDOMNode(DOM);
     TestUtils.Simulate.keyDown(contentScreen, { key: CONSTANTS.KEY_VALUES.ESCAPE });
     expect(toggleScreenCalled).toBe(true);
   });
 
+  it('should auto focus on first accessible element when specified', function() {
+    expect(document.activeElement).toBeFalsy();
+    var tree = TestUtils.renderIntoDocument(
+      <ContentScreen autoFocus={true} controller={ctrl}>
+        <AccessibleButton ariaLabel="fancy"></AccessibleButton>
```
Comment (aeng7) [inline tests/views/contentScreen-test.js]: too fancy for me

File: tests/components/accessibleButton-test.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+describe('AccessibleButton', function() {
+  var props, component;
+
+  function renderComponent() {
+    var tree = TestUtils.renderIntoDocument(<AccessibleButton {...props} />);
+    component = TestUtils.findRenderedComponentWithType(tree, AccessibleButton);
+  }
+
+  beforeEach(function() {
```
Comment (aeng7) [inline tests/components/accessibleButton-test.js]: Should we reset the "component" var here as well?

**Pull request merged**